corehq/apps/reports/standard/deployments.py

```diff
@@ -351,7 +368,8 @@ def process_rows(self, users, fmt_for_export=False):
                                     user.get('first_name', ''),
                                     user.get('last_name', '')),
                 _fmt_date(last_seen, fmt_for_export), _fmt_date(last_sync_date, fmt_for_export),
-                app_name or "---", build_version, commcare_version or '---'
+                app_name or "---", build_version, commcare_version or '---',
+                num_unsent_forms if num_unsent_forms is not None else "---",
```

Comment:
why not num_unsent_forms or "---"?

Reply:
because num_unsent_forms is an integer.. that means it'll be --- even if it's value is `0`

Reply:
I see.